pkg/config/config.go

```diff
@@ -106,6 +106,10 @@ func (wc *ControllerConfig) GetWebhooksEnabled() string {
 	return wc.GetPropertyOrDefault(webhooksEnabled, defaultWebhooksEnabled)
 }
 
+func (wc *ControllerConfig) GetNamespace() string {
```

Comment:
There's also [`k8sutil.GetOperatorNamespace`](https://godoc.org/github.com/operator-framework/operator-sdk/pkg/k8sutil#GetOperatorNamespace) that we use elsewhere -- may be more reliable.

Reply:
Yeah, thats what I originally was using but I didn't know if it made sense to constantly use that or do
>Just wondering if it makes sense to evaluate it once and return from Config object.

Reply:
I tried to play with it and it's what I've done https://github.com/JPinkney/che-workspace-operator/compare/fix-webhook-reference...sleshchenko:fix-webhook-reference-namespace-as-config?expand=1

I'm not fully happy with that since it seems all usage of `k8sUtils.GetOperatorNamespace` can't be replaced with `Config.GetOperatorNamespace` because init of config it not done as soon as possible...
But using ConfigMap's namespace it's not really the same as OperatorNamespace.
So, I'm OK with reverting to use K8sUtil and maybe later we'll refactor it in a nice and clear manner :crossed_fingers: